mimic/test/test_nova.py

```diff
@@ -4,6 +4,7 @@
 from __future__ import absolute_import, division, unicode_literals
 
 import json
+import random
```

Comment:
I don't think that we should be selecting the image randomly in the test case.  If we need to test with multiple images, we should explicitly select the different images being tested with and parameterize `test_create_image` so that we can have multiple test cases that call it with different images.  If the test becomes sensitive to image selection and fails in the case of certain images but not others, this will be very hard to diagnose.